From 9161ac30447a92aacb8c5a2f62d28c6d4192cb05 Mon Sep 17 00:00:00 2001
From: Orgad Shaneh <orgads@gmail.com>
Date: Thu, 13 Feb 2020 09:25:04 +0200
Subject: [PATCH 4/6] Python: Configure path fixes

---
 gdb/configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdb/configure b/gdb/configure
index 5361bf42952..2dfaf6d08c3 100755
--- a/gdb/configure
+++ b/gdb/configure
@@ -21838,7 +21838,7 @@ fi
 	as_fn_error $? "failure running python-config --ldflags" "$LINENO" 5
       fi
     fi
-    python_prefix=`${python_prog} ${srcdir}/python/python-config.py --exec-prefix`
+    python_prefix=`cygpath -u "$(${python_prog} ${srcdir}/python/python-config.py --exec-prefix)"`
     if test $? != 0; then
       have_python_config=failed
       if test "${with_python}" != auto; then
-- 
2.42.1

